lib/fluent/plugin/out_cloudwatch_logs.rb

```diff
@@ -120,6 +121,7 @@ def start
       options[:log_level] = :debug if log
       options[:region] = @region if @region
       options[:endpoint] = @endpoint if @endpoint
+      options[:ssl_verify_peer] = @ssl_verify_peer if @ssl_verify_peer
```

Comment:
According to https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudWatchLogs/Client.html it's `true` by default.
Although it seems that setting `false` is needed for your usage

> This is useful for testing against the https://github.com/spulec/moto AWS mock server, which by default runs with a self-signed certificate.

this code never set `false`.
Is it really the code that you expected?

Reply:
With this PR I meant to provide a conduit for setting ssl_very_peer in fluentd configuration. That's where I set it to false.

Reply:
I see what you mean now, my bad, fixed.